PRINCIPLES.md

```diff
@@ -158,3 +166,46 @@ _Principle: We help high-quality high-velocity cloud native open source projects
+
+#### Lead inclusively and empower all voices and opinions to be heard
+Actively seek to engage a variety of different voices and opinions as our community is built off a wide group of perspectives, experiences, and backgrounds. We are better as a community when we engage with all of the breadth of cloud native to better understand one another. As leaders, we are responsible for enabling everyone to not only have the opportunity to not only be engaged, but to bring balance to the discussions so no single voice - or perspective - overruns the desires of the collective whole. 
+
+For example, make an extra effort to ask the person who is new or who hasn’t yet spoken up in a meeting if they have any comments to offer. Make conscious effort to facilitate discussions in a manner that allows all voices to be heard and included, not just the loudest or most opinionated voices. Be aware that people who are shy, introverted, from historically underrepresented groups, and from certain cultures may be hesitant to speak up unless you clearly invite them to offer their opinion. Thus, make that extra effort to include people. Be sure to give everyone who wants to express their perspective an opportunity to offer their opinions even if they are not present in meetings. Invite people to continue sharing feedback asynchronously after meetings via Github, slack, or another channel that all participants have access to so they have the opportunity to provide comment.
+
+#### Foster respectful resolution
+
+Handle disagreements respectfully. Seek to resolve conflict and disagreement in a respectful and collaborative manner and encourage others to do the same. Show empathy, care, and concern for others’ feelings, perspectives, and experiences while working towards a resolution that considers the Community, the projects we interact with, the mission of our work, and what can reasonably be accomplished with consensus. For instance, a portion of a community presents a plan to completely refactor a project that dictates a stop work on all features. This could introduce contention and conflict within the community as many adopters and contributors are relying on those features. A reasonable resolution that may receive consensus from all members may be to refactor areas of the project where no feature development is occurring, complete outstanding features, and then refactor the remaining areas. Conflict and disagreement will happen, as a leader we must de-escalate, be the agent of calm, and provide neutrality for all parties to feel welcome in expressing their perspectives while pursuing an acceptable outcome. 
```

Comment:
```suggestion
Handle disagreements respectfully. Seek to resolve conflict and disagreement in a respectful and collaborative manner and encourage others to do the same. Show empathy, care, and concern for others’ feelings, perspectives, and experiences while working towards a resolution that considers the Community, the projects we interact with, the mission of our work, and what can reasonably be accomplished with consensus in accordance with the project's governance and scope. For instance, a portion of a community presents a plan to completely refactor a project that dictates a stop work on all features. This could introduce contention and conflict within the community as many adopters and contributors are relying on those features. A reasonable resolution that may receive consensus from all members may be to refactor areas of the project where no feature development is occurring, complete outstanding features, and then refactor the remaining areas. Conflict and disagreement will happen, as a leader we must de-escalate, be the agent of calm, and provide neutrality for all parties to feel welcome in expressing their perspectives while pursuing an acceptable outcome. 
```